tests/v2/test_peer_manager.nim

```diff
@@ -195,20 +195,14 @@ procSuite "Peer Manager":
       conn1Ok == false
 
       # Right after failing there is a backoff period
-      nodes[0].peerManager.peerStore.canBeConnected(
-        nonExistentPeer.peerId,
-        nodes[0].peerManager.initialBackoffInSec,
-        nodes[0].peerManager.backoffFactor) == false
+      nodes[0].peerManager.canBeConnected(nonExistentPeer.peerId) == false
 
     # We wait the first backoff period
-    await sleepAsync(2100.milliseconds)
+    await sleepAsync(2100)
```

Comment:
I think the better practice is to use the unit explicitly, i.e .seconds/.millis etc

Reply:
That makes sense, but when I try to use `milliseconds`, I get this error:

```
/home/vpavlin/devel/github.com/waku-org/nwaku/tests/v2/test_peer_manager.nim(201, 21) Error: type mismatch: got <TimeInterval>
but expected one of:
proc sleepAsync(duration: Duration): Future[void]
  first type mismatch at position: 1
  required type for duration: Duration
  but expression 'milliseconds(2100)' is of type: TimeInterval
proc sleepAsync(ms: int): Future[void]
  first type mismatch at position: 1
  required type for ms: int
  but expression 'milliseconds(2100)' is of type: TimeInterval

expression: sleepAsync(milliseconds(2100))
```

which is why I removed it

Reply:
Ok, I get the error now, glad I looked into Chronos a bit more today:D Fixed